<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>556</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="ledbtn">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>60</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>led</string>
   </property>
  </widget>
  <widget class="QPushButton" name="connectbtn">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>30</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>connect</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bhbtn">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>90</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>measurements</string>
   </property>
  </widget>
  <widget class="QPushButton" name="calculatebtn">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>260</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>calculate</string>
   </property>
  </widget>
  <widget class="QLabel" name="connect_your_device">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Connect your device:</string>
   </property>
  </widget>
  <widget class="QLabel" name="get_values">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Click to measure:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="values">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>140</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QSlider" name="brightslider">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>170</y>
     <width>241</width>
     <height>61</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="set_brightness_label">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>120</y>
     <width>181</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Adjust screen brightness:</string>
   </property>
  </widget>
  <widget class="QLabel" name="calculate_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>321</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Calculate and set the appropriate screen brightness:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sendbtn">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>340</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>send</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
